velox/connectors/hive/storage_adapters/gcs/GCSFileSystem.h

```diff
@@ -32,13 +32,33 @@ class GCSFileSystem : public FileSystem {
   // parameters.
   void initializeClient();
 
-  // Initialize a ReadFile using
-  // google::cloud::storage::Client::GetObjectMetadata
+  // Initialize a ReadFile
```

Comment:
Use /// for multi-line comments
https://github.com/facebookincubator/velox/blob/main/CODING_STYLE.md

Reply:
Fixed!